### PR TITLE
No icon for "Pipeline Argument Tab" #3436

### LINKED ISSUE #3436: No icon for "Pipeline Argument Tab"
No icon is shown for "Pipeline Argument Tab" Like, How the other tabs are shown.

image

### PR DESCRIPTION
**Pull request by lak-proddev**
Used "Dataflow_16.png" image temporarily. Until an appropriate image provides
fixes #3436

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java [hunk 1]
```diff
@@ -621,4 +622,10 @@ public void run() {
       }
     }
   }
+
+  @Override
+  public Image getImage() {
+    return DataflowUiPlugin.getDefault().getImageRegistry()
+        .get(DataflowUiPlugin.PIPELINE_TAB_IMAGE);
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java]: I've been thinking about this usage of the DataflowUiPlugin image registry.  This pipeline tab image is not frequently used.  `AbstractUIPlugin` advises that its [image registry should be kept to a minimum](https://git.eclipse.org/c/platform/eclipse.platform.ui.git/tree/bundles/org.eclipse.ui.workbench/Eclipse%20UI/org/eclipse/ui/plugin/AbstractUIPlugin.java?h=master#n109) as the images are not freed until the plugin is stopped (effectively when the application is exited).

This tab should allocate its own instance and dispose of it.

Comment (lak-proddev) [reply plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java]: That's what I did earlier right? :)

Comment (briandealwis) [reply plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java]: In #3437? But you didn't dispose of the image, right?  I'd be happy with https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/3437/commits/4cd6052b0a490046e5ff271d8e29888bf280a37c that kept the image in a field and disposed the field.

**Comment by lak-proddev** [comment]
Sorry :)
Seem to be done some mistake while rebasing. Created a new pull request to avoid confusion.
https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/3447

**Comment by chanseokoh** [comment]
Closing in favor of #3447.

**Pull request closed without merging**